_posts/2020-12-04-windows.md

```diff
@@ -65,21 +65,22 @@ and copy the Windows distributions there:
 Copy the following files to the distributions directories (for Windows 7 and newer):
```

Comment:
distributions directory (it's just one, isn't it?)

Reply:
I meant that you need to take the appropriate files for Win 10 and copy them to winos/Win10_EN-x64, and copy the files for Win 7 to winos/Win7_EN-x64. Files for Win 10 may not work for Win 7.

```
winos |
      + Win10_EN-x64
                     + pxeboot.n12
                     | bootmgr.exe
                       < .. >
                       
      + Win7__EN-x64
                     + pxeboot.n12
                     | bootmgr.exe
                       < .. >
```
Maybe this is a bad phrase or just need more clarification?